app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const app = express()
+const {getTopics} = require("./controllers/topics.controllers");
+const { psqlErrorHandler, customErrorHandler, serverErrorHandler } = require('./app-error-handlers')
+
+app.use(express.json())
+app.get('/api/topics', getTopics)
+
+app.all('*', (req, res, next) => {
+    res.status(404).send({message: 'path not found'})
+ })
```

Comment:
You have set up the error handler for path not found, in this case, please update your test to also test the message